archive/imo/imo2008_q2.lean

```diff
@@ -0,0 +1,156 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import data.real.basic
+import data.set.basic
+import data.set.finite
+
+/-!
+# IMO 2008 Q2
+(a) Prove that
+          ```x^2 / (x-1)^2 + y^2 / (y-1)^2 + z^2 / (z-1)^2 ≥ 1```
+for all real numbers x, y, z, each different from 1, and satisfying xyz = 1.
```

Comment:
Formulas / mathematical expressions are usually enclosed in backticks as well:
```suggestion
for all real numbers `x`, `y`, `z`, each different from 1, and satisfying `xyz = 1`.
```

Reply:
done!